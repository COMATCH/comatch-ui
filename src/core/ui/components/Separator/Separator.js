import React from 'react';

import './Separator.scss';

export const Separator = () => (<div className="Separator" />);