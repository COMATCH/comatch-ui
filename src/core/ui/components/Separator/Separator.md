
## Separator

From [`Separator.js`](Separator.js)



prop | type | default | required | description
---- | :----: | :-------: | :--------: | -----------



