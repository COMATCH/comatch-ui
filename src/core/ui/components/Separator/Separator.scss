@import '../../../styles/variables.scss';

.Separator {
    background-color: $co-color-mid-gray;
    height: 1px;
    margin: 20px 0;
}