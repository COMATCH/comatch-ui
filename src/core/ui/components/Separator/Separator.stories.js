import React from 'react';
import Highlight from 'react-highlight';
import { storiesOf } from '@storybook/react';
import { Panel } from '../Panel';
import { Separator } from './Separator';

storiesOf('Separator', module).add('Panel with Separators', () => (
    <>
        <Panel title="Just a panel" secondary>
            Some text here
            <Separator />
            Some more text
        </Panel>
        <Highlight className="html">
            {'<Panel title="Just a panel" secondary>\n' +
                '\tSome text here' +
                '\n\t<Separator />' +
                '\n\tSome more text' +
                '\n</Panel>'}
        </Highlight>
    </>
));
