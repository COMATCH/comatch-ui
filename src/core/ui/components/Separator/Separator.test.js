import React from 'react';
import renderer from 'react-test-renderer';
import { Separator } from './Separator';

describe('Separator', () => {
    it('should render correctly', () => {
        const separator = renderer.create(<Separator />).toJSON();
        expect(separator).toMatchSnapshot();
    });
});
