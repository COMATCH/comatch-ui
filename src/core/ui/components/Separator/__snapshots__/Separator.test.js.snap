// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should render correctly 1`] = `
<div
  className="Separator"
/>
`;
